### PR TITLE
fix: added _get_api_key from panasonic_cc

### PR DESCRIPTION
**Pull request by mortenf**
This seems to solve #103, based on a copy of the _get_api_key method from [panasonic_cc](https://github.com/sockless-coding/panasonic_cc/pull/336).

### CODE REVIEW

**Comment by lostfields** [comment]
Hi, I'm trying it out now but I get `pcomfortcloud.exceptions.ResponseError: (get_acc_client_id: Expected status code 200, received: 412: {"message":"Precondition Failed","code":41201}`. Not sure why, 

https://github.com/mortenf/python-panasonic-comfort-cloud/blob/f028a12c4bb46fbffd680cdededfe538e696c856/pcomfortcloud/authentication.py#L277

File: pcomfortcloud/authentication.py [hunk 1]
```diff
@@ -338,9 +364,7 @@ def _get_header_for_api_calls(self, no_client_id=False):
             "x-app-timestamp": timestamp,
             "x-app-type": "1",
             "x-app-version": self._app_version,
-            # Seems to work by either setting X-CFC-API-KEY to 0 or to a 128-long hex string
-            # "X-CFC-API-KEY": "0",
-            "x-cfc-api-key": generate_random_string_hex(128),
+            "x-cfc-api-key": self._get_api_key(timestamp, self._token["access_token"]),
```
Comment (lostfields) [inline pcomfortcloud/authentication.py:367]: I struggled a long time with this one, but the token sent into _get_api_key is wrong. After fixing that it's working for me.

```suggestion
            "x-cfc-api-key": self._get_api_key(timestamp, token_response["access_token"]),
```

File: pcomfortcloud/authentication.py [hunk 2]
```diff
@@ -242,7 +268,7 @@ def _get_new_token(self):
                 "x-app-timestamp": timestamp,
                 "x-app-type": "1",
                 "x-app-version": self._app_version,
-                "x-cfc-api-key": generate_random_string_hex(128),
+                "x-cfc-api-key": self._get_api_key(timestamp, self._token["access_token"]),
```
Comment (lostfields) [inline pcomfortcloud/authentication.py:271]: This should be fixed as well
```suggestion
                "x-cfc-api-key": self._get_api_key(timestamp, token_response["access_token"]),
```

Comment (mortenf) [reply pcomfortcloud/authentication.py]: Right, the new token should be used, where possible. However, I'm quite sure it should be the same access token as used in the Authorization header, so I've only changed one of the above.

**Comment by majurgens** [comment]
Nice. It works for me now. Thanks

File: pcomfortcloud/authentication.py (hunk 1 above)
Comment (lostfields) [reply pcomfortcloud/authentication.py:367]: Yeah, this shouldn't be changed

File: pcomfortcloud/authentication.py (hunk 2 above)
Comment (lostfields) [reply pcomfortcloud/authentication.py]: You are right. Great work

**Pull request merged**